types/ws/index.d.ts

```diff
@@ -85,43 +85,37 @@ declare class WebSocket extends EventEmitter {
     // HTML5 WebSocket events
     addEventListener(
         method: "message",
-        cb: (event: { data: any; type: string; target: WebSocket }) => void,
+        cb: (event: WebSocket.MessageEvent) => void,
         options?: WebSocket.EventListenerOptions,
     ): void;
     addEventListener(
         method: "close",
-        cb: (event: { wasClean: boolean; code: number; reason: string; target: WebSocket }) => void,
+        cb: (event: WebSocket.CloseEvent) => void,
         options?: WebSocket.EventListenerOptions,
     ): void;
     addEventListener(
         method: "error",
-        cb: (event: { error: any; message: any; type: string; target: WebSocket }) => void,
+        cb: (event: WebSocket.ErrorEvent) => void,
         options?: WebSocket.EventListenerOptions,
     ): void;
     addEventListener(
         method: "open",
-        cb: (event: { target: WebSocket }) => void,
+        cb: (event: WebSocket.Event) => void,
         options?: WebSocket.EventListenerOptions,
     ): void;
     addEventListener(method: string, listener: () => void, options?: WebSocket.EventListenerOptions): void;
```

Comment:
In `ws@8` this is a noop. The `handleProcotols` type change is already a breaking change so I wonder if it makes sense to remove this.

Reply:
Removed.